.github/workflows/backend-tests.yml

```
@@ -250,11 +256,20 @@ jobs:
           echo "Skipping backend tests because the PR only changed backend control-plane or tooling files."
 
       - name: Run tests (PR incremental)
-        if: github.event_name == 'pull_request' && env.SKIP_BACKEND_TESTS != '1'
+        if: github.event_name == 'pull_request' && env.SKIP_BACKEND_TESTS != '1' && env.FORCE_FULL_TESTS != '1'
         working-directory: src/api
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:28: Double quote to prevent globbing and word splitting

(shellcheck)